CONTRIBUTING.md

```diff
@@ -308,39 +308,57 @@ You can find an installation guide for all the different environments here:
 ## Product Management
 
 The ZITADEL Team works with an agile product management methodology.
-You can find all the issues prioritized and ordered in the [product board](https://github.com/orgs/zitadel/projects/1/views/2).
+You can find all the issues prioritized and ordered in the [product board](https://github.com/orgs/zitadel/projects/2/views/1).
 
 Every two weeks the team goes through all the new issues and decided about the priority, effort and if it is ready to start or in the backlog.
 If it is something critical or urgent we will have a look at it earlier.
-To show the community the needed information, each issue gets labels.
+To show the community the needed information, each issue gets attributes and labels.
 
-## About the Labels
+## About the attributes
 
-There are a few general labels that don't belong to a specific category.
+You can find the attributes on the project "Product Management".
 
-- **good first issue**: This label shows contibuters, that it is an easy entry point to start developing on ZITADEL.
-- **help wanted**: The author is seeking help on this topic, this may be from an internal ZITADEL team member or external contributors.
+### State
+
+The state should reflect the progress of the issue and what is going on right now.
+
+- **No status**: Issue just got added and has to be looked at.
+- **🧐 Investigating**: We are curently investigating to find out what the problem is, which priority it should have and what has to be implemented. Or we need some more information of the author.
+- **📨 Product Backlog**: If an issue is in the backlog, it is not currently being worked on. These are recorded so that they can be processed in the future. Issues with this state do not have to be completely defined yet.
+- **📝 Prioritized Product Backlog**: An issue with the state "Prioritized Backlog" is ready for the refinement from the perspective of the product owner (PO) to implement. This means the developer can find all the relevant information and acceptance criteria in the issue.
+- **🔖 Ready**: The issue is ready to take into a sprint. Difference to "prioritized..." is that the complexity is defined by the team.
+- **📋 Sprint Backlog**: The issue is scheduled for the current sprint.
+- **🏗 In progress**: Someone is working on this issue right now.
+- **👀 In review**: The issue is in review.
```

Comment:
When going from in progress to in review, do I have to request someone to review or is this done by PO? Would be maybe worth it to explicitly define the desired process.

Reply:
Also how external contributors should do it